src/test/features/response/draft/responseDraft.ts

```diff
@@ -190,69 +189,45 @@ describe('ResponseDraft', () => {
     })
   })
 
-  describe('isResponsePartiallyRejectedDueTo', () => {
+  describe('isResponsePartiallyAdmitted', () => {
 
-    it('should return false when no response type set', () => {
-      const draft: ResponseDraft = new ResponseDraft()
-      draft.response = undefined
-
-      expect(draft.isResponsePartiallyRejectedDueTo(RejectPartOfClaimOption.AMOUNT_TOO_HIGH)).to.be.eq(false)
-    })
-
-    it('should return error message when option argument is undefined', () => {
-      const draft: ResponseDraft = new ResponseDraft()
-      draft.response = new Response(ResponseType.PART_ADMISSION)
+    context('should return false when', () => {
 
-      expect(() => draft.isResponsePartiallyRejectedDueTo(undefined)).to.throw(Error, 'Option is undefined')
-    })
-
-    it('should return false message when response type is not a part rejection', () => {
-      ResponseType.except(ResponseType.PART_ADMISSION).forEach(responseType => {
+      it('no response set', () => {
         const draft: ResponseDraft = new ResponseDraft()
-        draft.response = new Response(responseType)
+        draft.response = undefined
 
-        expect(draft.isResponsePartiallyRejectedDueTo(RejectPartOfClaimOption.AMOUNT_TOO_HIGH)).to.be.equal(false)
+        expect(draft.isResponsePartiallyAdmitted()).to.be.eq(false)
```

Comment:
Any reason not to use `.to.be.false`?

Reply:
There are other chai chains available that could be used in this code; same question
`to.be.true`
`.to.be.undefined`
`.empty`
maybe others